typed-racket-more/typed/web-server/http.rkt

```diff
@@ -145,14 +156,14 @@
                         (-> String
                             Bytes
                             Request
-                            [#:timeout Number]
```

Comment:
This change isn't backwards-compatible so I think we should skip it.

Reply:
I'll update the PR momentarily. I've been conflicted about this change (e.g. <https://github.com/racket/web-server/pull/75#issuecomment-552677557>). As you say, it isn't strictly backwards-compatible, though for various reasons it seems unlikely to be a problem: in particular, <https://github.com/racket/typed-racket/commit/acf7707> had an analogous change from `Number` to `Real` that seems to have gone fine.

Reply:
The fact that this seems to come up quite often with `Number`/`number?` makes me wonder if we should think about what to call it in Rhombus …